x-pack/plugins/apm/public/components/fleet_integration/apm_agents/runtime_attachment/discovery_rule.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiBadge,
+  EuiPanel,
+  DraggableProvidedDragHandleProps,
+} from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { Operation } from '.';
+
+interface Props {
+  id: string;
+  order: number;
+  operation: string;
+  type: string;
+  probe: string;
+  providedDragHandleProps?: DraggableProvidedDragHandleProps;
+  onDelete: (discoveryItemId: string) => void;
+  onEdit: (discoveryItemId: string) => void;
+  operationTypes: Operation[];
+}
+
+export function DiscoveryRule({
+  id,
+  order,
+  operation,
+  type,
+  probe,
+  providedDragHandleProps,
+  onDelete,
+  onEdit,
+  operationTypes,
+}: Props) {
+  const operationTypesLabels = useMemo(() => {
+    return operationTypes.reduce((acc, current) => {
+      return {
+        ...acc,
+        [current.operation.value]: {
+          label: current.operation.label,
+          types: current.types.reduce((memo, { value, label }) => {
+            return { ...memo, [value]: label };
+          }, {}),
+        },
+      };
+    }, {} as { [operationValue: string]: { label: string; types: { [typeValue: string]: string } } });
+  }, [operationTypes]);
+
+  return (
+    <EuiPanel paddingSize="m">
+      <EuiFlexGroup>
```

Comment:
```suggestion
      <EuiFlexGroup alignItems="center">
```

Resolves an issue where the drag handle is not vertically centered.

<img width="914" alt="CleanShot 2021-12-06 at 13 04 21@2x" src="https://user-images.githubusercontent.com/4104278/144844538-bb14b5cd-098d-4384-ba9f-c0228672bfe2.png">

Now looks like

<img width="770" alt="CleanShot 2021-12-06 at 13 16 44@2x" src="https://user-images.githubusercontent.com/4104278/144844576-380050a1-2e4c-40ab-8bc2-53fb4552bff3.png">